content/en/synthetics/guide/email-validation.md

```diff
@@ -0,0 +1,68 @@
+kind: documentation
+description: Verify an email and its content with browser test steps
+further_reading:
+- link: "//synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+## Overview
+
+Classic web application journeys often involve emails being triggered and sent to a users’ mailbox, such as email verification after account creation, email sent to reset forgotten passwords, order confirmations, or mail confirmation after contact form submission.
+
+Making sure your application’s email mechanisms are working as expected is key to ensuring a great user experience on your website.
+
+Datadog Browser Tests allow you to:
+
+- Confirm an email was correctly sent following up on an in-app trigger
+- Check an email's content
+- Click on the links located within the sent emails in order to navigate to other URLs and validate your whole flows, including web and email steps
+
+To perform email validation inside a Datadog Browser test:
+
+## Create an email variable
+
+After creating your browser test, navigate to **Edit Recording**, select **Variables** and then select **Email** from the email variable dropdown list:
+
+{{< img src="synthetics/guide/email-validation/adding-variable.mp4" alt="Create an email variable" video="true"  width="100%">}}
+
+In the above example, an email variable called `EMAIL` is created. The newly created email variable generates a unique mailbox maintained by Datadog at every test execution. This is key to avoid any conflicts among your Browser test runs.
+
+## Record steps
+
+In the upper left corner of the UI, hit the **Start Recording** button and record the steps leading to the email being triggered using your freshly created email variable. Use the hand icon to record these steps.
+
+{{< img src="synthetics/guide/email-validation/record-steps.mp4" alt="Record your steps" video="true"  width="100%">}}
+
+The steps to fill the form are recorded and the click on the `Sign up` button triggers the email. An email is sent to the Datadog mailbox that was created for this recording session as a result (for example, `838-n3q-q2y.6238933596@synthetics.dtdg.co` in this example).
+
+## Confirm the email was sent
+
+You can now confirm that an email was sent. Click the **Assertion** button and select the `Test that an email was received` assertion.
+
+{{< img src="synthetics/guide/email-validation/assertion-step.mp4" alt="Add an assertion" video="true"  width="100%">}}
+
+If you want to make sure your email respects specific guidelines for its content, you can add more verifications against its subject and body.
+
+In the above example, the assertion is set to be successful if the email subject is `Welcome to Shopist!` and if its body contains the sentence `Your verification code is` and a verification code matches the `\d{1,6}` regex pattern.
```

Comment:
Link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions